c/common/test/rules/usageofassemblernotdocumented/test.c

```diff
@@ -0,0 +1,12 @@
+// COMPLIANT
```

Comment:
er sorry I asked this before but I forget the answer, was there any way to avoid having the exact same file 2x, like some way to ref the testcase from the other shared implementation? not a big deal if not!